rxnutils/chem/utils.py

```diff
@@ -336,14 +337,15 @@ def has_atom_mapping(
 
 
 def remove_atom_mapping(
-    smiles: str, is_smarts: bool = False, sanitize: bool = True
+    smiles: str, is_smarts: bool = False, sanitize: bool = True, canonical=True
```

Comment:
We forgot the type hint.

```suggestion
    smiles: str, is_smarts: bool = False, sanitize: bool = True, canonical: bool = True
```